desc/geometry/core.py

```diff
@@ -115,26 +113,30 @@ def N(self):
 
     @property
     def sym(self):
-        """stellarator symmetry"""
+        """Stellarator symmetry."""
         return self._sym
 
     @property
     @abstractmethod
     def grid(self):
-        """Default grid for computation"""
+        """Grid for computation."""
+
+    @abstractmethod
+    def change_resolution(self, L, M, N):
```

Comment:
would it be better to just have this take only keyword arguments? rather than having positional args for things that aren't used